File: CodingBat/Python/Warmup_2/array_front9.py
#Given an array of ints, return True if one of the first 4 elements in the arrya is a 9. The array length may be less than 4.
def array_front9(nums):
    if 9 in nums:
        if len(nums)<=4:
            return True
        else:
            for i in range(4):
                if nums[i]==9:
                    return True
            return False
    else:
        return False

File: CodingBat/Java/Warmup_2/arrayFront9.java
public class arrayFront9{
    
    public static boolean arrayFront9(int[] nums){
        // Given an array of ints, return true if one of the first 4 elements in the array is a 9. The array length may be less than 4.
        if(nums.length<=4){
            for(int i=0;i<nums.length;i++){
                if (nums[i]==9){
                    return true;
                }
            }
            return false;
        }
        else{
            for(int i=0;i<4;i++){
                if (nums[i]==9){
                    return true;
                }
            }
            return false;
        }
    }
    
    public static void main(String[] args) {
        
    }
}